pyproject.toml

```diff
@@ -20,7 +20,7 @@ build-backend = "setuptools.build_meta"
 name = "torch"
 description = "Tensors and Dynamic neural networks in Python with strong GPU acceleration"
 readme = "README.md"
-requires-python = ">=3.9,<3.14"
+requires-python = ">=3.9"
```

Comment:
We need to find another way to check `project.classifiers` includes the Python version classifiers properly (e.g., `Programming Language :: Python :: 3.14` is missing).

Or, we can remove all versioned classifiers `Programming Language :: Python :: 3.*` and only keep `Programming Language :: Python :: 3 :: Only`.

Reply:
Those classifiers are really inconsequential, they're only for info. So adding it manually when adding support for a new Python version and fixing it up separately if someone notices that one is missing should be okay, that's what every project I know of does.